bundles/fish.payara.eclipse.tools.server/src/fish/payara/eclipse/tools/server/deploying/PayaraServerBehaviour.java

```diff
@@ -324,7 +324,11 @@ public static String getVersion(PayaraServer server) throws PayaraIdeException {
 		Future<ResultString> future = ServerAdmin.exec(server, new CommandVersion());
 
 		try {
-			return future.get(30, SECONDS).getValue();
+			ResultString result = future.get(30, SECONDS);
+			if(!result.isAuth()) {
```

Comment:
Could you reformat this? some missing spaces for the if condition, probably something like:
```suggestion
			if (!result.isAuth()) {
```